examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,291 @@
+"""
+Multiphysics: Maxwell 3D - Icepak electrothermal analysis
+---------------------------------------------------------
+This example uses PyAEDT to set up a simple Maxwell design consisting of a coil and a ferrite core.
+Coil current is set to 100A, and coil resistance and ohmic loss are analyzed.
+Ohmic loss is mapped to Icepak, and a thermal analysis is performed. Icepak calculates a temperature distribution,
+and it is mapped back to Maxwell (2-way coupling). Coil resistance and ohmic loss are analyzed again in Maxwell.
+Results are printed in AEDT Message Manager.
+Keywords: Litz wire model, Icepak coupling, 2-way coupling
```

Comment:
I don't remember ever seeing a "Keywords" entry in examples. Is this something new PyAEDT is doing?

Reply:
Hi @PipKat!
The idea is to have a series of keywords for each example in order to be able to identify them easily. 
We haven't done anything yet so I think we can remove them (for now) from the example.
@Samuelopez-ansys we could think about the feasibility of this new feature.

Reply:
@gmalinve Right, remove it, and we can think how to add it in the new PyAEDT examples repository (in progress)

Reply:
@gmalinve @Samuelopez-ansys I think the concept is awesome. If such a feature is implemented, I think it should be more prominent--perhaps a bolded section right after a brief description of the example. I'll leave you guys to work out the details!